### PR TITLE
[price-pusher] add workflow to push docker image

### PR DESCRIPTION
**Pull request by 0xfirefist**

### CODE REVIEW

File: .github/workflows/price-pusher-image-push.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      - uses: aws-actions/configure-aws-credentials@8a84b07f2009032ade05a88a28750d733cc30db1
+        with:
+          role-to-assume: arn:aws:iam::192824654885:role/github-actions-ecr
+          aws-region: eu-west-2
+      - uses: docker/login-action@v2
+        with:
+          registry: public.ecr.aws
+        env:
+          AWS_REGION: us-east-1
+      - uses: aws-actions/amazon-ecr-login@v1
```
Comment (0xfirefist) [inline .github/workflows/price-pusher-image-push.yml:39]: do we need to do ecr_login? I see in price server file we didn't, but in price pusher file in other repo we did. @ali-bahjati

Comment (ali-bahjati) [reply .github/workflows/price-pusher-image-push.yml:39]: Hmm I don't know. @thmzlt knows better.

**Review by ali-bahjati — APPROVED** [review]
Can you also update the dockerfiles?

**Comment by 0xfirefist** [comment]
> Can you also update the dockerfiles?

I have updated them in a previous PR.

**Pull request merged**